src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` should be in the `model` package, not in the `db` package, as it represents a domain model rather than database-related code.